src/components/Buildbaron/BuildBaronContent.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import styled from "@emotion/styled";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { useParams } from "react-router-dom";
+import { Banners } from "components/Banners";
+import {
+  useBannerDispatchContext,
+  useBannerStateContext,
+} from "context/banners";
+import { BuildBaron } from "gql/generated/types";
+import { withBannersContext } from "hoc/withBannersContext";
+import { BBTitle, TitleAndButtons } from "./BBComponents";
+import { BBFileTicket, CreatedTickets } from "./BBFIleTicket";
+import { BuildBaronTable } from "./BuildBaronTable";
+
+interface Props {
```

Comment:
nit `BuildBaronCoreProps` since there are multiple components in this file

Reply:
thanks!